experiments/tf_trainer/tf_hub_classifier/run.py

```diff
@@ -41,7 +35,7 @@ def serving_input_fn_tfrecords():
         name="input_example_tensor"
     )
     feature_spec = {
-        text_feature_name: tf.FixedLenFeature([], dtype=tf.string),
+        'text': tf.FixedLenFeature([], dtype=tf.string),
```

Comment:
This one too.. `base_model.TEXT_FEATURE_KEY` ?

Reply:
Done.